### PR TITLE
[rom_ctrl,doc] Weaken description of what pwrmgr is expected to do

### PR DESCRIPTION
**Pull request by rswarbrick**

### CODE REVIEW

**Comment by rswarbrick** [comment]
Sorry, @tjaychen! I assigned this to you, rather than adding you as a reviewer (whatever that means). Anyway, it now looks like I intended :-/

**Comment by tjaychen** [comment]
sounds good, i should push to this one also and add some language on the `pwrmgr` side.

**Review by tjaychen — APPROVED** [review]

File: hw/ip/rom_ctrl/doc/_index.md [hunk 1]
```diff
@@ -114,7 +114,8 @@ Since the code in ROM is the first thing to execute, an attacker that modifies i
 As such, OpenTitan needs some form of ROM integrity checking and the ROM checker is the module in charge of providing it.
 
 After bringing the ROM controller module out of reset, the power manager must wait until `pwrgr_data_o.done` is asserted before starting the host processor.
-In PROD state, it will also check that `pwrmgr_data_o.good` is `On`: if not, it will refuse to boot.
+The ROM controller also passes the `pwrmgr_data_o.good` signal.
```
Comment (matutem) [inline hw/ip/rom_ctrl/doc/_index.md:117]: FYI: This intersects with https://github.com/lowRISC/opentitan/pull/9773

Comment (rswarbrick) [reply hw/ip/rom_ctrl/doc/_index.md:117]: Thanks! I'll have a look there in a minute.

**Pull request merged**